src/main/resources/scripts/advanced god mode.sk

```diff
@@ -0,0 +1,36 @@
+#
+# Toggleable god mode
+#
+
+options:
+    permission: command.god # the permission required to use /god
+    disable on join: true # if god mode should be disabled when the user joins
+    disallow pvp: true # if users in god mode shouldn't be allowed to pvp
+
+    enabled: &7Enabled god mode. # when the user enables god mode
+    disabled: &7Disabled god mode. # when the user disables god mode
+    currently enabled: &7You have god mode enabled so you can't hit other users. # when the user tries to pvp in god mode while <disallow pvp> is true
+
+on join:
+    if {@disable on join} is true:
+        delete {god::%player%}
+        send "{@disabled}" to player
+
+on damage of player:
+    if {@disallow pvp} is true:
+        {god::%attacker%} is set
+        send "{@currently enabled}" to attacker
+        cancel event
+    if {god::%victim%} is set:
+    	cancel event
+
+command /god <offline player=%player%>:
+    description: Toggle god mode
+    permission: {@permission}
+    trigger:
+        if {god::%{_p}%} is not set:
```

Comment:
{_p} has never been set, it should be changed to arg-1, same with the rest

Reply:
yeah I meant to do that